tools/generate_property_translations.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+#
+# Generates translation strings from gecko property files for lupdate
+# to process
+
+import os, os.path, sys, re, dataclasses, functools
+
+PREFIX = "sailfish_components_webview_popups-la"
```

Comment:
This is correct prefix... but the script itself should be pushed to the sailfish-components-webview repository. Similarly under tools I'd say.

Will give a spin for this one already.

Reply:
Right, thanks! I opened a new PR to the correct repository.